core/utils/load_batch.py

```diff
@@ -138,42 +146,9 @@ def check_format_date(sample_batch_data):
         try:
             sample_batch_data[column] = pd.to_datetime(
                 sample_batch_data[column]
-            ).dt.strftime("%Y-%m-%d %H:%M:%S")
+            ).dt.strftime("%Y-%m-%d")
         except Exception:  # Unknown string format: string present at position x
-            error_cause = core.core_config.ERROR_DATE_FORMAT_FIELD.copy()
-            return " ".join(error_cause)
-
-
-def format_date(sample_batch_data):
```

Comment:
I would keep two separate functions for checking and formating the date, because it is confusing for one function to return two different things depending on the validation. In case you want to merge them to reduce code, i would follow a similar approach to this: 
```
def function(prr):
if errorx:
  return false, "error cause string"
else:
  return true, ""

fun_check, error_cause = function(prr)
```
where you return both the true/false if it is checked, and the formatted date in case the format is validated.